pkg/ccl/backupccl/backup_job.go

```diff
@@ -331,7 +331,16 @@ func backup(
 			// Collect all the table stats for this table.
 			tableStatisticsAcc, err := statsCache.GetTableStats(ctx, tableDesc.GetID())
 			if err != nil {
-				return RowCount{}, err
+				// Successfully backed up data is more valuable than table stats that can
+				// be recomputed on restore, and so if we fail to collect the stats of a
+				// table we do not want to mark the job as failed.
+				// The lack of stats on restore could lead to suboptimal performance when
+				// reading/writing to this table until the stats have been recomputed.
+				wrappedErr := errors.Wrap(err,
+					fmt.Sprintf("failed to collect stats for table %d during a backup",
```

Comment:
nit: Let's also include the table name here for readability (in addition to the table ID)

Reply:
done.